### DISCUSSION TITLE
grain boundry

### DISCUSSION THREAD

**Discussion by lyg1232** [mtex-toolbox/mtex#1519] [Ask Anything] [answered]
Hello everyone
 now，i have a problem and need your help. I have ebsd data of a magnesium alloy. How can I get the grain boundaries with different colors as shown in the picture?
![27e6ee6d15b18934a05bc578e0718cd](https://user-images.githubusercontent.com/81472625/182329854-cea78748-e8d7-40de-b5eb-cca279212080.jpg)

**Comment by kilir** [comment]
Hi,
also here I suggest to have a look in the documentation. 
Cheers,
Rüdiger

**Comment by lucapbelotti** [comment, marked as answer]
Hi,

As Rüdiger suggested, this is well documented in: https://mtex-toolbox.github.io/BoundaryPlots.html

Best,

Luca

**Comment by Liujianhua00** [comment]
I did similar work, recently, you can try the following code:
```matlab
gB = grains.boundary('phase1','phase1');
plot(gB,gB.misorientation.angle./degree,'linewidth',1,'displayName','phase1-phase1 gbs',gB.misorientation.angle./degree);
```
